### PR TITLE
try to correct the  pmv and ppd calucaltion while the unit transferri…

### PR DESCRIPTION
**Pull request by karlliu666**
…ng into fahrenheit and ft/s to ensure the ourcome maintaining at all

### CODE REVIEW

**Comment by karlliu666** [comment]
problem2

problem1

The flaw fixing is ongoing

File: components/show_results.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
     ElementsIDs,
 )
 
+def feet_per_second_to_meters_per_second(ftps: float) -> float:
```
Comment (FedericoTartarini) [inline components/show_results.py:13]: this function is already defined in the code, you should use that and not define it again

File: components/show_results.py [hunk 2]
```diff
@@ -24,23 +26,39 @@ def display_results(inputs: dict):
         if selected_model == Models.PMV_ashrae.name:
             standard = "ashrae"
 
+        # Retrieve input temperature values.
+        tdb_get = inputs[ElementsIDs.t_db_input.value]
+        tr_get = inputs[ElementsIDs.t_r_input.value]
+        vr_get=inputs[ElementsIDs.v_input.value]
```
Comment (FedericoTartarini) [inline components/show_results.py:32]: format the code with Black

File: components/show_results.py [hunk 3]
```diff
@@ -24,23 +26,39 @@ def display_results(inputs: dict):
...
+        met_get = inputs[ElementsIDs.met_input.value]
+
+        # If the user selects imperial units, convert them to the international standard.
+        if units == UnitSystem.IP.value:
```
Comment (FedericoTartarini) [inline components/show_results.py:36]: all this code is not needed since the input dict should be in Celsius already

**Review by FedericoTartarini — COMMENTED** [review]
All this is not needed since the inputs should not be in IP units the opposite code is needed. When the user selects IP we need to convert the output to IP units

**Pull request closed without merging**